CMakeLists.txt

```diff
@@ -210,15 +207,6 @@ install(FILES "${CMAKE_SOURCE_DIR}/Contributors.md"
 install(FILES "docs/scap-security-guide.8"
     DESTINATION "${CMAKE_INSTALL_MANDIR}/man8")
 
-if (UNIX AND SSG_COMPATIBILITY_SYMLINK)
-    install(CODE "
-        file(MAKE_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/xml/scap/ssg/)
-    ")
-    install(CODE "
-        execute_process(COMMAND ln -sf ../../../scap/ssg ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/xml/scap/ssg/content)
-    ")
-endif()
-
```

Comment:
@mpreisler why remove this code? If SSG_COMPATIBILITY_SYMLINK is FALSE, it doesn't create the symlink, so then, the code shouldn't have to be removed, right?